protocol/x/bridge/app_test.go

```diff
@@ -142,6 +142,7 @@ func TestBridge_Success(t *testing.T) {
 						genesisState.SafetyParams = tc.safetyParams
 					},
 				)
+				genesis.GenesisTime = tc.blockTime
```

Comment:
@tqin7 The `blockTime` in the test case is ambiguous. I believe setting genesis time with this time best suits the intention of this test (so the BlockTime is already set to `tc.blockTime` by the time bridge module looks at the bridge events

Reply:
Yeah that works!